features/channel/managing_channels/editing_channel.feature

```diff
@@ -33,3 +33,11 @@ Feature: Editing channel
     Scenario: Seeing disabled base currency field during channel edition
         When I want to modify a channel "Web Channel"
         Then the base currency field should be disabled
+
+    @ui @todo
+    Scenario: Changing type of channel
```

Comment:
In this scenario, there is missing information about the channel type before change